<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunsTableWidget</class>
 <widget class="QWidget" name="RunsTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="qf::qmlwidgets::TableViewToolBar" name="tblRunsToolBar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Class start:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblClassStart">
          <property name="styleSheet">
           <string notr="true">font-weight: bold</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>interval</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblClassInterval">
          <property name="styleSheet">
           <string notr="true">font-weight: bold</string>
          </property>
          <property name="text">
           <string>---</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableView" name="tblRuns"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TableView</class>
   <extends>QTableView</extends>
   <header location="global">qf/qmlwidgets/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::TableViewToolBar</class>
   <extends>QWidget</extends>
   <header location="global">qf/qmlwidgets/tableviewtoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
